airbyte-integrations/connectors/source-oracle/src/test/java/io/airbyte/integrations/source/oracle/OracleJdbcSourceAcceptanceTest.java

```diff
@@ -115,6 +131,107 @@ void cleanUpTables() throws SQLException {
       conn.close();
   }
 
+  @Override
+  protected List<AirbyteMessage> getTestMessages() {
```

Comment:
just curious: why do we need to override this method now

Reply:
All methods in JdbcSourceAcceptanceTest work for all of our sources. I could change this in JdbcSourceAcceptanceTest, but then all tests for sources that don't use microseconds will not work correctly. Therefore, for some sources, I have redefined the methods you specified for the time being. When we update all our sources to work with microseconds, then we can update JdbcSourceAcceptanceTest and remove the overrides from the sources. While this is relevant only for oracle, other Jdbc sources successfully cope with the conversion.